hoomd/md/pytest/test_methods.py

```diff
@@ -0,0 +1,20 @@
+import hoomd
+import pytest
+import numpy
+import itertools
+
+def attach_and_sim(simulation_factory, two_particle_snapshot_factory, method):
```

Comment:
Can someone comment on the best way to write this test so that I can easily plug in each integration method and have the test run on it? I'm not that familiar with pytest, and while this will get the job done, I think there is probably a better way.

Reply:
Use a parameterized fixture for the method: https://docs.pytest.org/en/latest/fixture.html#parametrizing-fixtures